### ISSUE TITLE
python doesnt find revChatGPT module

### ISSUE THREAD

**Issue by PowermateEx** [Zero6992/chatGPT-discord-bot#207]
It seems like the new version cant find the revChatGPT Module. i installed the requirements etc, it seems to be working only in visual studio code, but not if i want to use it as a cmd window.

**Comment by Zero6992** [comment]
Please provide the error message.

**Comment by PowermateEx** [comment]
Just removed the full path from the error, so its better to read.
```
Traceback (most recent call last):
  File "\main.py", line 1, in <module> from src import bot
  File "\src\bot.py", line 4, in <module> from src import responses
  File "\src\responses.py", line 1, in <module> from revChatGPT.V1 import AsyncChatbot
ModuleNotFoundError: No module named 'revChatGPT.V1'
```

**Comment by Zero6992** [comment]
You need to run `pip install -r requirements.txt` in the directory where the 'requirements.txt' file is located

**Comment by PowermateEx** [comment]
Yes, i did that. I've read the installation tutorial. The version before seemed to work just fine, now i did a clean one, but it doesnt work on cmd. It does work on vsc for some reason though, as i already said. Did you try running it on your system through cmd with python?

**Comment by Zero6992** [comment]
Everything is okay on my environment

Run `pip freeze` in your terminal to check the version of revChatGPT

**Comment by PowermateEx** [comment]
```
aiohttp==3.8.4
aiosignal==1.3.1
anyio==3.6.2
async-timeout==4.0.2
asyncio==3.4.3
attrs==22.2.0
certifi==2022.12.7
charset-normalizer==3.0.1
discord.py==2.1.0
frozenlist==1.3.3
h11==0.14.0
httpcore==0.16.3
httpx==0.23.3
idna==3.4
multidict==6.0.4
OpenAIAuth==0.3.2
python-dotenv==0.21.1
requests==2.28.2
revChatGPT==2.3.10
rfc3986==1.5.0
sniffio==1.3.0
urllib3==1.26.14
yarl==1.8.2
```
Seems to be alright

**Comment by PowermateEx** [comment]
Maybe im missing something.
You can add me on Discord if you want.
#Powermate1744

**Comment by PowermateEx** [comment]
I now removed everything and made a complete fresh installation of python and all the modules. Now i get a new error:
```
Traceback (most recent call last):
  File "...\main.py", line 1, in <module> from src import bot
  File "...\src\bot.py", line 4, in <module> from src import responses
  File "...\src\responses.py", line 9, in <module> chatbot = AsyncChatbot(config={"email":openAI_email, "password":openAI_password, "session_token":session_token})
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 558, in __init__
    super().__init__(
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 147, in __init__
    self.__check_credentials()
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 161, in __check_credentials
    self.__login()
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 271, in __login
    self.__refresh_headers(auth.access_token)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 183, in __refresh_headers
    self.__cache_access_token(email, access_token)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 230, in __cache_access_token
    self.__write_cache(cache)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 45, in wrapper
    out = func(*args, **kwargs)
  File "C:\Users\neona\AppData\Roaming\Python\Python310\site-packages\revChatGPT\V1.py", line 234, in __write_cache
    os.makedirs(osp.dirname(self.cache_path), exist_ok=True)
  File "c:\users\neona\appdata\local\activestate\cache\5381dc1f\lib\os.py", line 225, in makedirs
    mkdir(name, mode)
FileNotFoundError: [WinError 3] The system could not find path: ''
```

**Comment by eighttran** [comment]
> 

same problem

**Comment by Zero6992** [comment]
@PowermateEx  @eighttran  you can refer here for some information 

https://github.com/acheong08/ChatGPT/issues/935

**Comment by PowermateEx** [comment]
Bug apparently fixed now in version 2.3.15 and 2.3.16.
Otherwise it works setting the Home environment up to this directory "C:\Users\neona\AppData\Local"

**Comment by eighttran** [comment]
> @PowermateEx @eighttran you can refer here for some information
> 
> [acheong08/ChatGPT#935](https://github.com/acheong08/ChatGPT/issues/935)

thanks for reply, I solved it
